dbConn.js

```diff
@@ -0,0 +1,9 @@
+require('dotenv').config();
+const Knex = require('knex');
+const config = require('./knexfile');
+
+/* Initialize knex. */
+const environment = process.env.NODE_ENV;
+const knex = Knex(config[environment]);
+
+module.exports = knex;
```

Comment:
Please get rid of all the red things.

Reply:
deleted dbConn file. and moved connection to index